gsoc/2020/mentors.md

```diff
@@ -5,6 +5,9 @@ layout: plain
 
 ## Full Mentor List (Name, Email, Org)
 
+* Christian Ariza [christian.ariza@gmail.com](mailto:christian.ariza@gmail.com) CERN
 * Ulrik Egede [ulrik.egede@monash.edu](mailto:ulrik.egede@monash.edu) Monash University
+* Valentin Kuznetsov [vkuznet@gmail.com](mailto:vkuznet@gmail.com) Cornell University, USA
```

Comment:
Just noticed that there are mentors not only from CERN, but also from Cornell and INFN. If the project is a collaboration, you need to add the corresponding organizations as: _gsocorgs/2020/cornell.md and infn.md respectively (see also comment on the organization field in your proposal)